src/Alexa.Net/project.json

```diff
@@ -6,11 +6,11 @@
     "Newtonsoft.Json": "9.0.1"
   },
   "frameworks": {
-    "netstandard1.4": {}
+    "netstandard1.2": {}
```

Comment:
Also the frameworks changed to 1.2, but the dependencies still have .NETStandard.Library 1.6 -- I'm still trying to wrap my head around all the different standard/lib/core aspects that are changing in .NET.